docs/src/operate/configuration/index.md

```diff
@@ -5,8 +5,171 @@ sidebar_position: 2
 description: How to configure %%te%%
 ---
 
-import DocCardList from '@theme/DocCardList';
+The settings of all the %%te%% components are grouped in the `/etc/tedge/tedge.toml` file, using [TOML](https://toml.io/).
+These configuration settings are organized in a hierarchy that reflects the component hierarchy.
+For instance, all the settings related to Cumulocity IoT share a `c8y` prefix, such as `c8y.url` for the Cumulocity URL.
 
-How to configure %%te%%
+This file can be edited directly and can even be extended to include plugin-specific settings.
+However, it's recommended to use the [`tedge config`](../../references/cli/tedge-config.md) command
+to edit the settings as it provides guidance for expected settings and checks for invalid entries.
 
-<DocCardList />
+## Common Commands
+
+The following is a list of common commands which can be used to get/set/list %%te%% configuration.
+
+### List configuration with descriptions
+
+Display a complete list of available settings with their purpose.
+
+```sh
+tedge config list --doc
+```
+
+### List configuration that have been set or have defaults
+
+List the settings for which a specific value has been set.
+
+```sh
+tedge config list
+```
+
+### Get a single configuration value
+
+Display the value for the `c8y.url` setting, if one has been set.
+
+```sh
+tedge config get c8y.url
+```
+
+### Set configuration value
+
+Update/set the value for the `c8y.url` setting.
+
+```
+tedge config set c8y.url mytenant.cumulocity.com`
+```
+
+### Reset a configuration value to use default value
+
+Unset any user-specific value for the `c8y.url` setting, using then the default value.
+
+```sh
+tedge config unset c8y.url
+```
+
+## Example
```

Comment:
Make example a subsection so it is easier to extend in the future, and it is easier for users to find via the page's TOC

```suggestion
### Examples

### Change path used for temporary files
```